website/docs/r/app_service.html.markdown

```diff
@@ -321,6 +329,24 @@ A `ip_restriction` block supports the following:
 
 ---
 
+---
+
+A `scm_ip_restriction` block supports the following:
+
+* `ip_address` - (Optional) The IP Address used for this IP Restriction in CIDR notation.
+
+* `virtual_network_subnet_id` - (Optional) The Virtual Network Subnet ID used for this IP Restriction.
+
+-> **NOTE:** One of either `ip_address` or `virtual_network_subnet_id` must be specified
+
+* `name` - (Optional) The name for this IP Restriction.
+
+* `priority` - (Optional) The priority for this IP Restriction. Restrictions are enforced in priority order. By default, priority is set to 65000 if not specified.
+
```

Comment:
The service does not set default priority for restrictions if any restriction item has a priority explicitly set, this can be addressed with the schema change suggestion above.